index.html

```diff
@@ -109,10 +109,11 @@ <h3>
                         <span class="carrot"></span>
                         <div class="device-box-header">HTC Vive</div>
                         <span>
-                            Works with <a href="https://webvr.rocks/firefox">Firefox Nightly</a>,
-                            <a href="https://blog.mozvr.com/webvr-servo-architecture-and-latency-optimizations/">Servo</a>, and
+                            Works with <a href="https://webvr.rocks/firefox">Firefox</a>,
+                            <a href="https://blog.mozvr.com/webvr-servo-architecture-and-latency-optimizations/">Servo</a> and
                             <a href="https://www.supermedium.com">Supermedium</a>
-                            on Windows
+                            on Windows.
+                            <a href="https://nightly.mozilla.org/">Firefox Nightly</a> on macOS.
```

Comment:
did you intend to add a `<br>` before this line?

Reply:
also, you can use the Oculus Rift w/ Firefox via SteamVR (OpenVR) with macOS, right? I'm pretty certain it's not just the Vive. cc @blairmacintyre @kearwood?

Reply:
Added `<br>`. 
Re `Oculus` I wasn't sure about it. Waiting for their answer